__tests__/resource/csharp-nunit-input.json

```diff
@@ -0,0 +1,251 @@
+{
+  "sentAt": {
+    "low": 1105186912,
+    "high": 395,
+    "unsigned": true
+  },
+  "serverInfo": {
+    "apiUrl": "https://api.kobiton.com",
+    "portalUrl": "https://portal.kobiton.com",
+    "username": "kobitonadmin+appiumScriptGen",
+    "apiKey": "your_kobiton_api_key"
+  },
+  "isManualSession": true,
+  "manualSessionId": "605",
+  "devices": [
+    {
+      "id": 1881,
+      "name": "Lenovo Tab M10 FHD Plus",
+      "capabilities": {
+        "platformName": "Android",
+        "platformVersion": "10",
+        "resolution": {
+          "width": 720,
+          "height": 1600,
+          "scale": 1
+        }
+      }
+    }
+  ],
+  "testSteps": [
+    {
+      "id": "17000",
+      "context": "NATIVE",
+      "selectorConfigurations": [
+        {
+          "selectors": [
+            {
+              "type": "xpath",
+              "value": "//*[@resource-id='com.android.chrome:id/url_bar']"
+            },
+            {
+              "type": "className",
+              "value": "android.widget.EditText"
+            },
+            {
+              "type": "xpath",
+              "value": "//android.widget.EditText[@resource-id='com.android.chrome:id/url_bar']"
+            },
+            {
+              "type": "xpath",
+              "value": "//android.widget.EditText[@text='edition.cnn.com']"
+            }
+          ]
+        }
+      ],
+      "actionJson": "{\"command\":\"touchOnElement\",\"x\":\"0.387\",\"y\":\"0.430\",\"value\":\"beyoung.in\"}",
```

Comment:
on an `touchOnElement ` we cannot have "value" field, please keep it as a separated test step with command "sendKey", sample: https://github.com/kobiton/appium-script-generator/blob/master/__tests__/resource/java-junit-input.json#L63